tools/check/check.go

```diff
@@ -366,10 +366,12 @@ func populateConfig() error {
 }
 
 func main() {
-	logger.Init("", *verbose, false, os.Stderr)
+	logger.Init("", false, false, os.Stderr)
 	flag.Parse()
 	cmdline.Parse("auto")
+	logger.SetLevel(logger.Level(*verbose))
```

Comment:
Is this referring to the bool verbose flag which is now replaced by the int verbosity flag?